flake.nix

```
@@ -122,6 +124,61 @@
         };
         formatter = pkgs.alejandra;
 
+        checks = let
+          nixos-lib = import "${nixpkgs}/nixos/lib" {};
```

```suggestion
          nixpkgs-lib = nixpkgs.lib;
```

lib doesn't require an import and nixos-lib is typical something different https://github.com/NixOS/nixpkgs/tree/master/nixos/lib

---

This doesn't have the `runTest` attribute; I think this really might need the nixos/lib import?

---

Derp, I've overread this 😂 You're right